### PR TITLE
reviewing-the-audit-log-for-your-organization: Fix links and correct typo

### LINKED ISSUE #21539: Broken links and typo in reviewing-the-audit-log-for-your-organization.md
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

- File: https://github.com/github/docs/blob/main/content/organizations/keeping-your-organization-secure/managing-security-settings-for-your-organization/reviewing-the-audit-log-for-your-organization.md
- Live: https://docs.github.com/en/organizations/keeping-your-organization-secure/managing-security-settings-for-your-organization/reviewing-the-audit-log-for-your-organization

### What part(s) of the article would you like to see updated?

- The `#repository_secret_scanning_custom_pattern-category-actions` link is broken
- The `#repository_secret_scanning_push_protection`  link is broken
- `Secret` is spelled incorrectly

The links should be fixed and the typo should be corrected.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by rvrvrv**
### Why:
 - `repository_secret_scanning_custom_pattern` link is broken
 - `repository_secret_scanning_push_protection` link is broken
 - `Protecting pushes with secret scanning` has typo

Closes https://github.com/github/docs/issues/21539

### What's being changed (if available, include any code snippets, screenshots, or gifs):
image

### Check off the following:

- [x] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by Rambo250000000000** [comment]
https://www.paypal.com/pools/c/8OmsxaiDuZ

**Comment by cmwilson21** [comment]
@rvrvrv Thanks for submitting this PR, providing great information in the template, and for linking it to your issue! ✨ 

I'll get this triaged for review! ⚡

File: content/organizations/keeping-your-organization-secure/managing-security-settings-for-your-organization/reviewing-the-audit-log-for-your-organization.md [hunk 1]
```diff
@@ -77,8 +77,8 @@ To search for specific events, use the `action` qualifier in your query. Actions
...
-| [`repository_secret_scanning_push_protection`](#respository_secret_scanning_push_protection) | Contains repository-level activities related to secret scanning custom patterns. For more information, see "[Protecting pushes with secert scanning](/code-security/secret-scanning/protecting-pushes-with-secret-scanning)." {% endif %}
+| [`repository_secret_scanning_custom_pattern`](#repository_secret_scanning_custom_pattern-category-actions) | Contains repository-level activities related to secret scanning custom patterns. For more information, see "[Defining custom patterns for secret scanning](/code-security/secret-scanning/defining-custom-patterns-for-secret-scanning)." {% endif %}{% ifversion secret-scanning-audit-log-custom-patterns %}
+| [`repository_secret_scanning_push_protection`](#repository_secret_scanning_push_protection) | Contains repository-level activities related to secret scanning custom patterns. For more information, see "[Protecting pushes with secret scanning](/code-security/secret-scanning/protecting-pushes-with-secret-scanning)." {% endif %}
```
Comment (skedwards88) [inline content/organizations/keeping-your-organization-secure/managing-security-settings-for-your-organization/reviewing-the-audit-log-for-your-organization.md:81]:
```suggestion
| [`repository_secret_scanning_push_protection`](#repository_secret_scanning_push_protection-category-actions) | Contains repository-level activities related to secret scanning custom patterns. For more information, see "[Protecting pushes with secret scanning](/code-security/secret-scanning/protecting-pushes-with-secret-scanning)." {% endif %}
```

**Review by skedwards88 — COMMENTED** [review]
Thanks for fixing this! I just have one small change to fix the second link. I'll go ahead and incorporate that, then we'll get this merged down for you.

**Review by skedwards88 — APPROVED** [review]

**Pull request merged**